install.rst

```diff
@@ -23,475 +18,39 @@ Confirm you will be able to provision the following hardware to replicate the re
-4. Download `any version of the Mattermost Server <https://docs.mattermost.com/administration/upgrade.html#version-archive>`_ by typing:
-
-   -  ``wget https://releases.mattermost.com/X.X.X/mattermost-X.X.X-linux-amd64.tar.gz``
-   -  Where ``vX.X.X`` is typically the latest Mattermost release version, which is currently ``v3.3.0``. 
-   
-5. Unzip the Mattermost Server by typing:
-
-   -  ``tar -xvzf mattermost-X.X.X-linux-amd64.tar.gz``
-
-6. Create the storage directory for files. We assume you will have
-   attached a large drive for storage of images and files. For this
-   setup we will assume the directory is located at
-   ``/mattermost/data``.
-
-   -  Create the directory by typing:
-   -  ``sudo mkdir -p /mattermost/data``
-   -  Set the ubuntu account as the directory owner by typing:
-   -  ``sudo chown -R ubuntu /mattermost``
-
-7. Configure Mattermost Server by editing the config.json file at
-   ``/home/ubuntu/mattermost/config``
-
-   -  ``cd ~/mattermost/config``
-   -  Edit the file by typing:
-   -  ``vi config.json``
-   -  replace ``DriverName": "mysql"`` with ``DriverName": "postgres"``
-   -  replace
-      ``"DataSource": "mmuser:mostest@tcp(dockerhost:3306)/mattermost_test?charset=utf8mb4,utf8"``
-      with
-      ``"DataSource": "postgres://mmuser:mmuser_password@10.10.10.1:5432/mattermost?sslmode=disable&connect_timeout=10"``
-   -  Optionally you may continue to edit configuration settings in
-      ``config.json`` or use the System Console described in a later
-      section to finish the configuration.
-
-8. Test the Mattermost Server
-
-   -  ``cd ~/mattermost/bin``
-   -  Run the Mattermost Server by typing:
-   -  ``./platform``
-   -  You should see a console log like ``Server is listening on :8065``
-      letting you know the service is running.
-   -  Stop the server for now by typing ``ctrl-c``
-
-9. Setup Mattermost to use the Upstart daemon which handles supervision
-   of the Mattermost process.
-
-   -  ``sudo touch /etc/init/mattermost.conf``
-   -  ``sudo vi /etc/init/mattermost.conf``
-   -  Copy the following lines into ``/etc/init/mattermost.conf``
-
-      ::
-
-          start on runlevel [2345]
-          stop on runlevel [016]
-          respawn
-          limit nofile 50000 50000
-          chdir /home/ubuntu/mattermost
-          setuid ubuntu
-          exec bin/platform
-
-   -  You can manage the process by typing:
-   -  ``sudo start mattermost``
-   -  Verify the service is running by typing:
-   -  ``curl http://10.10.10.2:8065``
-   -  You should see a page titles *Mattermost - Signup*
-   -  You can also stop the process by running the command
-      ``sudo stop mattermost``, but we will skip this step for now.
+Follow the "Setup Mattermost Server" instructions in our `Production Ubuntu Install Guide <https://docs.mattermost.com/install/prod-ubuntu.html#set-up-mattermost-server>`_ to setup your mattermost server.
+
```

Comment:
This is not about optimization, it's about making this document less confusing. If we just copy the stuff from the other documents people will be confused (like Shift) about if these install instructions are different and just not follow them. 

Also having a bunch of nonsense in this document obscures the point of the document which is the performance optimizations.